docs/pages/material-ui/api/rating.json

```diff
@@ -57,7 +57,11 @@
       "iconActive",
       "decimal"
     ],
-    "globalClasses": { "disabled": "Mui-disabled", "focusVisible": "Mui-focusVisible" },
+    "globalClasses": {
+      "readOnly": "Mui-readOnly",
```

Comment:
This is a breaking change we will need to manually add the `MuiRating-readOnly` in the Rating component, and add a comment to remove it in v6.

Reply:
This diff could help:

```diff --git a/packages/mui-material/src/Rating/Rating.js b/packages/mui-material/src/Rating/Rating.js
index f0f2234850..de1020c23c 100644
--- a/packages/mui-material/src/Rating/Rating.js
+++ b/packages/mui-material/src/Rating/Rating.js
@@ -499,7 +499,14 @@ const Rating = React.forwardRef(function Rating(inProps, ref) {
       ref={handleRef}
       onMouseMove={handleMouseMove}
       onMouseLeave={handleMouseLeave}
-      className={clsx(classes.root, className)}
+      className={clsx(
+        classes.root,
+        {
+          // TODO v6: remove this class as it duplicates with the gloabl state class Mui-readOnly
+          'MuiRating-readOnly': readOnly,
+        },
+        className,
+      )}
       ownerState={ownerState}
       role={readOnly ? 'img' : null}
```

Reply:
Even with the suggested changes, it doesn't change the outcome of the rating.json file. I'm not sure what to do here.

Reply:
I pushed a fix, there was a spelling mistake in the `useUtilityClasses`.

Reply:
@ZeeshanTamboli would you like to finish this up and merge it?

Reply:
Sure, I'll do it.